### PR TITLE
fix Northridge material file

### LINKED ISSUE #35: Strange material properties of Northridge scenario
Hi all, 

I am trying to simulate the Northridge scenario and I am a bit confuse about the material properties. Is it really the case that the Lame parameters reach `~39 tera Pa` at the 30 km depth?

https://github.com/SeisSol/Examples/blob/1a068e494ecb0d193cfbba65185a63d98e0f09b0/Northridge/northridge_material.yaml#L10-L21

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
fixes #35 based on Duo's inputs

### CODE REVIEW

File: Northridge/createRhoMuLambda.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+def compute_nu(rho):
+    if rho < 2060.0:
+        return 0.40
+    elif rho > 2500.0:
+        return 0.25
+    else:
+        return 0.40 - ((rho - 2060.0) * 0.15 / 440.0)
+
+
+def compute_rho_mu_lambda(Vp):
```
Comment (sebwolf-de) [inline Northridge/createRhoMuLambda.py:14]: But also computes quality factors, right?

**Review by sebwolf-de — COMMENTED** [review]
It puzzles me that not only the "wrong parameters" i.e. the ones in the range of Tera Pascals are changed, but also the density a shallow depth.

File: Northridge/createRhoMuLambda.py (hunk 1 above)
Comment (Thomas-Ulrich) [reply Northridge/createRhoMuLambda.py]: yes. I've updated.

**Comment by Thomas-Ulrich** [comment]
> 

about the density, I don't know, the original file was just wrong.

**Review by sebwolf-de — APPROVED** [review]

**Comment by AliceGabriel** [comment]
Thanks. Pretty embarrassing. I assume we need to update the training image?

**Pull request merged**